NEWS.md

```diff
@@ -46,6 +46,7 @@ OpenSSL 3.1
     FIPS provider, TLS 1.2 will, by default, mandate the use of an
     extended master secret and the Hash and HMAC DRBGs will not operate
     with truncated digests.
+  * Enable extra Arm64 optimization on Windows for GHASH, RAND and AES.
```

Comment:
I don't think this is the right place for this

Reply:
Yeah, it needs to go into the 3.2 section and furthermore this is not significant enough IMO to be in NEWS.md -> put it inside CHANGES.md instead.

Reply:
Sure, it has been moved to CHANGES.md